.github/workflows/main.yml

```diff
@@ -126,7 +128,8 @@ jobs:
         # the next command might be needed for foxy distro, since this package is not installed
         # by default in ubuntu 20.04. For other distro, the apt install command will be ignored.
         sudo apt install -y ros-${{matrix.ros_distro}}-sensor-msgs-py
-        python3 src/realsense-ros/realsense2_camera/scripts/rs2_test.py non_existent_file
+        source ../.venv/bin/activate
```

Comment:
Why do we need this source command?

Reply:
I think to activate the virtual env (venv) created in line 116.
when creating venv, we should activate it to "run" inside it.
every command after it will be treated as inside the virtual env.
@Arun-Prasad-V  am I right ?

Reply:
yes, to use python in virtual env.